docs/user-guide/user-guide.rst

```diff
@@ -1260,6 +1260,70 @@ Click the Create Asset button to create an Asset manually. Enter relevant Asset
 
 -------------------------------
 
+Create Asset Batch
+~~~~~~~~~~~~~~~
+
+Example CSV File that can be used to create multiple Assets at once:  
+:download:`Download Sample Asset CSV </user-guide/Stig-Manager-Asset-Batch-Import.csv>`
+
+The uploaded CSV must follow this format:
+
+.. list-table:: 
+   :header-rows: 1
+
+   * - Column Name
+     - Description
+     - Data Requirements
+   * - Name
+     - The name of the asset
+     - (Required) Must be between 1 and 255 characters long. 
+   * - Description
+     - Description of the asset
+     - (Optional) 255 characters max, any characters over 255 will be truncated
+   * - IP
+     - The IP address of the asset
+     - (Optional) 255 characters max, any characters over 255 will be truncated
+   * - FQDN
+     -  Fully Qualified Domain Name of the asset
+     - (Optional) 255 characters max, any characters over 255 will be truncated
+   * - MAC
+     - The MAC address of the asset
+     - (Optional) 255 characters max, any characters over 255 will be truncated
+   * - Non-Computing
+     - Indicates if the asset is a non-computing device
+     - (Optional) Will default to FALSE if not provided. Valid values are TRUE or FALSE.
+   * - STIGs
+     - The STIGs to be assigned to the asset
+     - (Optional) Newline separated list of STIG BenchmarkIds. STIG installed in STIG Manager before they can be assigned to an Asset.
+   * - Labels
+     - The labels to be assigned to the asset
+     - (Optional) Newline separated list of Label Names. Labels in the file will be created if they do not already exist in the Collection.
+   * - Metadata
+     - The metadata to be assigned to the asset
+     - (Optional) JSON-encoded object for key-value metadata. Only one layer deep is supported. The keys and the values must be strings. The keys must be unique within the object.
+
+
+Click the Import Assets CSV to create an Asset from a CSV File. 
+
+.. thumbnail:: /assets/images/collection-manage-csv-batch-create.png
+      :width: 50% 
+      :show_caption: True
+      :title: Select 'Import Assets CSV' button
+
+
+Begin by selecting a CSV that contains asset data. Once uploaded, the system parses each row, validating the entries before proceeding. Only rows with valid data are considered for submission — any invalid rows are automatically ignored for submission with the respective errors on that row shown. During validation, the system checks for existing labels referenced in the CSV and automatically creates any missing ones. After validation completes, the user can submit the data. Submission is only enabled if at least one valid asset is detected.

```

Comment:
During validation, labels are not created, they are just marked to be created if the user proceeds.